RELEASE_NOTES

```diff
@@ -3,9 +3,46 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyconfig provides 4,596 easyconfig files, for 737 different software packages,
+The latest version of easybuild-easyconfig provides 4,818 easyconfig files, for 778 different software packages,
 34 different (compiler) toolchains, 12 software bundles and 1 meta-package.
 
+v2.5.0 (December 17th 2015)
+---------------------------
+
+feature + bugfix release
+- add GCCcore easyconfig that can be used as base for all compilers (without getting in the way) (#2214)
+    - + easyconfig for GCC/4.9.3-2.25: bundle of GCCcore 4.9.3 and binutils 2.25
+- added example easyconfig files for 39 new software packages:
+    DIRAC (#2212), GeoIP (#2172, #2185), GeoIP-C (#2172, #2185), graph-tool (#1591), gtkglext (#2217),
+    Intel Advisor (#2210), InterProScan (#2225, #2227, #2234), intltool (#2136), kallisto (#2173), LibUUID (#1930),
+    LuaJIT (#2153), libXcursor (#2136), libXrandr (#2136), libXtst (#2143), libdap (#1930), libtasn1 (#2208),
+    libxkbcommon (#2136), MRtrix (#2217, #2218), MultiNest (#2166, #2168), Nipype (#2150), PPfold (#2183, #2187),
+    p11-kit (#2208), pangox-compat (#2217), Qt5 (#2136), randrproto (#2136), rhdf5 (#2175), Stampy (#2180, #2182),
+    scikit-umfpack (#2061), scp (Python pkg) (#2196), sleuth (#2175), traits (#2150), vincent (#2169, #2185),
+    XKeyboardConfig (#2136), xcb-util (#2136), xcb-util-image (#2136), xcb-util-keysyms (#2136),
+    xcb-util-renderutil (#2136), xcb-util-wm (#2136), zlibbioc (#2175)
+- added new easyconfigs for existing toolchains:
+    intel/2015.08 (#2194), intel/2016.00 (#2209), intel/2016.01 (#2219), iomkl/2015.03 (#2155)
+- added additional easyconfigs for various supported software packages: version updates, different toolchains, ...
+    including CMake 3.4.1, HDF5 1.8.16, netCDF 4.3.3.1, netCDF-Fortran 4.4.2, numpy 1.10.1, Octave 4.0.0,
+              OpenFOAM 3.0.0, Python 2.7.11
+- various other enhancements, including:
+    - add tidyr to R 3.2.1 easyconfigs (#2174)
+    - enable C++ support in MIGRATE-N (#2178)
+    - also installed shared libraries for AMD and UMFPACK in SuiteSparse (#2061)
+    - fix software name for ParaView (was: Paraview) (#2132)
+    - enable building of shared libraries for binutils (#2133)
+    - harden binutils built with dummy toolchain by linking to system libraries via RPATH (#2228)
+    - enhance easyconfig unit tests to check that each easyconfig file is in the right subdirectory (#2232)
+- various bug fixes, including:
+    - fix ALADIN patch file to not use relative paths, and adjust list of ALADIN sources accordingly (#2207, #2213)
+    - rename patch files for OpenFOAM to be in line with other patches (#2226)
+    - fix typo in bzip2 source URLs (#2204)
+    - correctly link ncurses for libreadline (#2206)
```

Comment:
`correctly` is not a good word choice. More like: `Force linking of ncurses in libreadline`.